<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CopterSimGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="copterTableView">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>PGP</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="paramPgpBox">
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>AGP</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDoubleSpinBox" name="paramAgpBox">
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>PGI</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="paramPgiBox">
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>AGI</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="paramAgiBox">
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>PGD</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="paramPgdBox">
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>AGD</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QDoubleSpinBox" name="paramAgdBox">
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>VGP</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="paramVgpBox">
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Max Corr</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QDoubleSpinBox" name="paramMaxCorrBox">
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>VGI</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="paramVgiBox">
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Max Tilt</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QDoubleSpinBox" name="paramMaxTiltBox">
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>VGD</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDoubleSpinBox" name="paramVgdBox">
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="paramSetButton">
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="paramGetButton">
            <property name="text">
             <string>Get</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addCopterButton">
            <property name="text">
             <string>Add Copter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="addXBox">
            <property name="minimum">
             <double>-99.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="addYBox">
            <property name="minimum">
             <double>-99.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="scenario1Button">
            <property name="text">
             <string>Scenario 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="scenario2Button">
            <property name="text">
             <string>Scenario 2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="scenario3Button">
            <property name="text">
             <string>Scenario 3</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="deleteCopterButton">
            <property name="text">
             <string>Delete Copter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deletAllCoptersButton">
            <property name="text">
             <string>Delete all copters</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="crashButton">
            <property name="text">
             <string>Crash Simulation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="resetSimulationButton">
            <property name="text">
             <string>Reset simulation</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="saveSpeedBox">
            <property name="text">
             <string>Log speeds</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="speedSampleLabel">
            <property name="text">
             <string>0 samples</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearSpeedButton">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveSpeedFileButton">
            <property name="text">
             <string>Save to file...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="collisionLabel">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>No collision</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>FaultCheck Probe</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Identifier</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="fcOffsetButton">
              <property name="text">
               <string>Offset</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Duration</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="fcDurationBox">
              <property name="maximum">
               <number>10000</number>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="fcAmpButton">
              <property name="text">
               <string>Amp</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Delay</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="fcDelayBox">
              <property name="maximum">
               <number>10000</number>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="fcSetToButton">
              <property name="text">
               <string>Set to</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Value</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="fcValueBox">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>500.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QPushButton" name="fcBitFlipButton">
              <property name="text">
               <string>Bit Flip</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="0" colspan="4">
             <widget class="QPushButton" name="fcResetButton">
              <property name="text">
               <string>Reset</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QComboBox" name="fcIdBox"/>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="fcUpdateIdButton">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>FaultCheck Packet</string>
           </attribute>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Joystick axes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="joystickA1Bar">
          <property name="minimum">
           <number>-32768</number>
          </property>
          <property name="maximum">
           <number>32767</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="format">
           <string>%v</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="joystickA2Bar">
          <property name="minimum">
           <number>-32768</number>
          </property>
          <property name="maximum">
           <number>32767</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="format">
           <string>%v</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="joystickA3Bar">
          <property name="minimum">
           <number>-32768</number>
          </property>
          <property name="maximum">
           <number>32767</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="format">
           <string>%v</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="joystickA4Bar">
          <property name="minimum">
           <number>-32768</number>
          </property>
          <property name="maximum">
           <number>32767</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="format">
           <string>%v</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Joystick</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="joystickPortEdit">
        <property name="text">
         <string>/dev/input/js0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="joystickConnectedLabel">
        <property name="text">
         <string>Not Connected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="disconnectJoystickButton">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="connectJoystickButton">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="joystickSendBox">
        <property name="text">
         <string>Send joystick</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>893</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
